spec/shared_examples.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+SUPPORT_TABLES = %w[
```

Comment:
I would suggest keeping shared examples here only if they are actually used from multiple other spec files. In particular, I would suggest moving `SUPPORT_TABLES` and  `RSpec.shared_examples "nonempty #{table} response"` to `support_table_spec.rb`.